docs/claim-transparency.md

```diff
@@ -0,0 +1,345 @@
+
+## The Claim Format
+
+We propose the following structure, based on the [in-toto Statement standard](https://github.com/in-toto/attestation/blob/main/spec/README.md#statement),
+for specifying security and privacy claims. This format is meant to be generic and allow specifying
+different types of claims.
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [{ ... }],
+
+  "predicateType": "https://github.com/project-oak/transparent-release/schema/claim/v1",
+  "predicate": {
+    "issuer": {
+      "id": "<URI>"
+    },
+    "claimType": "<URI>",
+    "metadata": {
+      "issuedOn": "<TIMESTAMP>",
+      "validFrom": "<TIMESTAMP>",
+      "expiresOn": "<TIMESTAMP>",
+    },
+    "claimSpec": { /* object */ },
+    "evidence": [
+      {
+        "role": "<STRING>",
+        "uri": "<URI>",
+        "digest": { /* DigestSet */ }
+      }
+    ]
+  }
+}
+```
+
+### Fields
+
+This section describes the semantics of each field in the claim format:
+
+- **subject** _(array of objects, required)_:
+  Set of artifacts that the claim applies to. Usually only one artifact (e.g., source code, or some binary).
+  - **subject[*].digest** and **subject[*].name** as defined by Statement in the in-toto standard.
+- **issuer** _(object, required)_:
+  Identifies the entity (person, organization, or an automated tool) that reviewed/audited/examined
+  the artifact in the subject of this claim.
+  - **issuer.id** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
+    URI indicating the issuer’s identity (e.g., mailto:issuer@example.com).
+- **claimType** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
+  URI indicating what type of claim was issued. It determines the meaning of claimSpec and evidence below.
+- **metadata** _(object, required)_:
+  Additional details about the claim.
+
+  - **metadata.issuedOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp of when the assessment of the artifact was performed. In most cases, the
+    assessment involves manually auditing or reviewing some source code. The timestamp in this case
+    does not have to be the accurate moment the review process started, but only the date on which
+    the review was done.
+  - **metadata.validFrom** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp from which the claim is effective, and the artifact is endorsed for use. In most
+    cases, it is the same as issuedOn timestamp, but can be any time after that.
+  - **metadata.expiresOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp of when the artifact is no longer endorsed for use. This, combined with the
```

Comment:
Inclusive? Note that the standard (and IMO reasonable) way of expressing start / end validity is to use `notBefore` / `notAfter` (e.g. x509 and JWT use similar terminology already). Also I think there is a difference between issuedOn (which is really metadata, and probably not really checked by anything in practice, so it belongs where it is now), vs the validity period, which is really expected to be checked every time. So for that perhaps a separate `validity` object would be useful, with guidance on what **must** (as per  https://www.ietf.org/rfc/rfc2119.txt ) be checked by verifiers.

Reply:
Feel free to create a PR and suggest the correction once this is merged.